utils.py

```diff
@@ -1075,8 +1085,14 @@ def label_multiline(layout, text='', icon='NONE', width=-1, max_lines=10, split_
     li = 0
     for l in lines:
         urls = []
+
+        # if 3>l.find(' - ')>-1:
```

Comment:
Add-on is full of dead code, would be great to not add more but try to remove it instead. So if it is useful I would implement it now, or remove it now.